.github/workflows/build-and-deploy.yml

```diff
@@ -415,5 +417,5 @@ jobs:
           arm-access-key: ${{ secrets.ARM_ACCESS_KEY }}
           azure-credentials: ${{ secrets.AZURE_CREDENTIALS }}
           environment: ${{ matrix.environment }}
-          sha: ${{ needs.build.outputs.IMAGE_TAG }}
+          sha: ${{ needs.test.outputs.IMAGE_TAG }}
```

Comment:
do we need something similar in review apps building?

Reply:
The deploy review app job depends on the build job directly so it's fine